target/consensource-ui
# Fix non-sticky test banner footer css

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Fixes broken CSS with the test banner.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

View http://localhost:8080/index_factory.html#!/signUp and confirm that the banner stays stuck at the bottom of the page while scrolling.
